src/madhuraMJun20/MethodsOfCharClass.java

```diff
@@ -0,0 +1,36 @@
+package madhuraMJun20;
+
+public class MethodsOfCharClass {
+	
+	void findCharFreq(String name) {
+		int digitC=0 ; int letterC =0; int upperCase=0; int lowerCase = 0; int spChar=0;
+		
+		for(int index=0;index<name.length();index++) {
+			
+			char ch= name.charAt(index);
+			if(Character.isDigit(ch))
+				digitC++;
+			else if (Character.isLetter(ch)) {
+				letterC++;
+				if(Character.isUpperCase(ch))
```

Comment:
letterC this variable is not needed dear. 
Think here, you can reduce else if > if and else condition.

Reply:
Done.